gn3/api/ctl.py

```diff
@@ -1,22 +1,24 @@
 """module contains endpoints for ctl"""
 
-# so small 
-
 from flask import Blueprint
 from flask import request
 from flask import jsonify
 
 from gn3.computations.ctl import call_ctl_script
 
-ctl = Blueprint("ctl",__name__)
+ctl = Blueprint("ctl", __name__)
+
 
-@ctl.route("/run_ctl",methods=["POST"])
+@ctl.route("/run_ctl", methods=["POST"])
 def run_ctl():
-	"""endpoint to run ctl"""
-	ctl_data = request.json
+    """endpoint to run ctl
+    input: request form object
+    output:json object enum::(response,error)
```

Comment:
**nitpick**: With end-points, I usually see no point adding docstrings; unless it's something complicated.  Very much IMHO this one.